web/styles/dark_theme.css

```diff
@@ -466,6 +487,18 @@
         color: inherit;
     }
 
+    .dropdown-list-search .dropdown-list-search-input:focus {
+        background-color: hsl(225deg 6% 7%);
+        border: 1px solid hsl(0deg 0% 100% / 50%);
+        box-shadow: 0 0 5px hsl(0deg 0% 100% / 40%);
+    }
+
+    #bot_owner_dropdown_widget,
+    #compose_select_recipient_widget {
+        border-color: hsl(0deg 0% 0% / 60%);
+        background-color: hsl(0deg 0% 0% / 20%);
+    }
```

Comment:
Could this be replaced by just having a `.dropdown-list-button` item just under `select` in the block 2 above here?

Reply:
Yeah, but I don't like `color: inherit;` there, which is unpredictable. Although, used `.dropdown-widget-button` class here.